README.md

```diff
@@ -210,6 +210,7 @@ func main() {
 | form    | String | Deprecated. URL encoded request body. This will also set the `Content-Type` header to `application/x-www-form-urlencoded` |
 | headers | Table  | Additional headers to send with the request |
 | timeout | Number/String | Request timeout. Number of seconds or String such as "1h" |
+| auth    | Table  | Basic auth for request. Set user and pass keys. |
```

Comment:
Can you describe what this value should look like here?

Reply:
Maybe with an example.

Reply:
Hello.
This value should look like: `auth={user="user", pass="pass"}`
Full example: `response, error_message = http.request("GET", "http://example.com", {query="page=1", auth={"user="user", pass="pass"}})`